apps/vx-mark/frontend/package.json

```diff
@@ -82,10 +84,12 @@
     "@types/styled-components": "^5.1.7",
     "@votingworks/ballot-encoder": "workspace:*",
     "@votingworks/fixtures": "workspace:*",
+    "@votingworks/grout": "workspace:*",
     "@votingworks/logging": "workspace:*",
     "@votingworks/types": "workspace:*",
     "@votingworks/ui": "workspace:*",
     "@votingworks/utils": "workspace:*",
+    "@votingworks/vx-mark-backend": "workspace:*",
```

Comment:
We should be able to move this to dev deps and only use `import type` to pull in types from the backend